system_tests/resources/linux/network-name-blueprint.yaml

```diff
@@ -0,0 +1,137 @@
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml
+  - http://www.getcloudify.org/spec/fabric-plugin/1.3/plugin.yaml
+  - ../../../plugin.yaml
+
+inputs:
+  template_name:
+    description: >
+      Template to clone VMs from
+    default: WindowsServer2012R2ByCloudifyDefault
+  external_network:
+    description: >
+      Which external network to deploy the VM on
+    default: VM Network
+  external_network_distributed:
+    description: >
+      Whether the external network is on a distributed switch
+    default: true
+  management_network:
+    description: >
+      Which management network to deploy the VM on
+    default: VM Network
+  management_network_distributed:
+    description: >
+      Whether the management network is on a distributed switch
+    default: true
+  vsphere_username:
+    type: string
+    description: >
+      User login for vsphere
+  vsphere_password:
+    type: string
+    default: ''
+    description: >
+      User password for vsphere
+  vsphere_host:
+    description: >
+      vSphere host
+    type: string
+  vsphere_port:
+    description: >
+      port which vCenter Server system uses to monitor data transfer
+      from SDK clients
+    default: 443
+    type: string
+  vsphere_datacenter_name:
+    description: >
+      datacenter name
+    default: Datacenter
+    type: string
+  vsphere_resource_pool_name:
+    description: >
+      Resource pool name. Represents a set of physical resources:
+      a single host, a subset of a host's resources, or resources
+      spanning multiple hosts
+    default: Resources
+    type: string
+  vsphere_auto_placement:
+    description: >
+      signifies if server is to be automatically placed on a host
+    default: false
+    type: boolean
+  test_network_name:
+    description: >
+      The name to give to the new test network
+    type: string
+    default: systestnetwork
+  test_network_connect_name:
+    description: >
+      The name used to connect to the created network.
+    default: systestnetwork
+  test_network_vswitch:
+    description: >
+      The vswitch to create the network on.
+    default: DVSwitch
+  test_network_vlan:
+    description: >
+      The VLAN of the new network.
+    default: 404
+
+node_types:
+  connection_configuration:
+     derived_from: cloudify.nodes.Root
+     properties:
+       connection_config: {}
+
+node_templates:
+  connection_configuration:
+    type: connection_configuration
```

Comment:
I think this should be done using `dsl_definitions` instead of abusing `node_types` to create a non-type-type